src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -920,4 +921,164 @@ public static void createSourceBQUpdateTable() throws IOException, InterruptedEx
     PluginPropertyUtils.addPluginProp(" bqTargetTable",  bqTargetTable);
     BeforeActions.scenario.write("BQ Target Table " +  bqTargetTable + " updated successfully");
   }
+
```

Comment:
Already created hooks for existing source and sink table can be used from the already raised PR for BQ, if we have the same schema int,float,string and boolean.So the duplicity can be avoided